### PR TITLE
[14.0][MIG] account_asset_comment

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

`account_asset_management` dependency added in https://github.com/nuobit/odoo-addons/pull/56.

### CODE REVIEW

**Comment by ChrisOForgeFlow** [comment]
@MiquelRForgeFlow @eantones I suggest depreciate this module, because on standard version there's a field called note

https://github.com/OCA/account-financial-tools/blob/ccd65daa77357097fe5698ca2ac77b0b75d21b8d/account_asset_management/models/account_asset.py#L96

**Comment by eantones** [comment]
I agree but we need a way to migrate the data from "comment" to "note"

**Comment by ChrisOForgeFlow** [comment]
I do not think that a migration script is necessary, since the field is called the same, you have to verify it in the migration process or testing the migrated base, but the type of field and the name is the same as far as I check, the only thing that change is the field label

**Comment by eantones** [comment]
The new field is called "note" and the one in this module is called "comment", this is the actual field name, not the label.  I don't see your point here.

**Comment by ChrisOForgeFlow** [comment]
You are right I see here

https://github.com/nuobit/odoo-addons/blob/c34c4680c6492343d0907bed608f5ae6b8afc49a/account_asset_comment/models/models.py#L11

Migration scripts is right here, I'll test migration script to check functional review

**Review by mariadforgeflow — APPROVED** [review]
Code review.

File: account_asset_comment/models/models.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Eric Antones <eantones@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class AccountAsset(models.Model):
+    _inherit = "account.asset"
+
+    note = fields.Text(string="Comment")
```
Comment (eantones) [inline account_asset_comment/models/models.py]: @MiquelRForgeFlow  It's not necessary to rename the field, the default "Note" name is fine, you can remove this.

File: account_asset_comment/migrations/14.0.1.0.0/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        """
+        UPDATE account_asset
+        SET note = CASE WHEN note IS NULL OR note = '' THEN comment
+            WHEN comment IS NOT NULL THEN COALESCE(note, '') || comment
+            ELSE note END"""
+    )
+
+
+def migrate(cr, version):
+    comment_to_note(cr)
```
Comment (eantones) [inline account_asset_comment/migrations/14.0.1.0.0/pre-migration.py:20]: @MiquelRForgeFlow After the migration from "comment" to "note", the old "comment" field should be removed.

**Review by eantones — CHANGES_REQUESTED** [review]

File: account_asset_comment/migrations/14.0.1.0.0/pre-migration.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply account_asset_comment/migrations/14.0.1.0.0/pre-migration.py:20]: I am not a fan of deleting columns, but I am not opposed to it. So column "comment" deleted.

**Comment by MiquelRForgeFlow** [comment]
pre-commit fixed in https://github.com/nuobit/odoo-addons/pull/163.

**Pull request merged**